apps/full-site-editing/full-site-editing-plugin/posts-list-block/class-posts-list-block.php

```diff
@@ -32,6 +32,7 @@ class Posts_List_Block {
 	private function __construct() {
 		add_action( 'init', array( $this, 'register_blocks' ), 100 );
 		add_action( 'enqueue_block_editor_assets', array( $this, 'enqueue_scripts' ), 100 );
+		add_action( 'enqueue_core_site_editor_assets', array( $this, 'enqueue_scripts' ), 100 );
```

Comment:
Interesting - I would expect this to be included via `enqueue_block_editor_assets`. Was that not the case?